lib/flip_card.dart

```diff
@@ -37,7 +37,8 @@ class AnimationCard extends StatelessWidget {
         }
         return Transform(
           transform: transform,
-          alignment: Alignment.center,
+          filterQuality: FilterQuality.none,
+          alignment: FractionalOffset.center,
```

Comment:
Why is this necessary?
As per [documentation](https://api.flutter.dev/flutter/painting/FractionalOffset-class.html), the information described by this is the same.